src/main/host/descriptor/mod.rs

```diff
@@ -32,6 +32,17 @@ bitflags::bitflags! {
     }
 }
 
+impl FileStatus {
+    pub fn as_o_flags(&self) -> libc::c_int {
+        self.bits() as i32
+    }
+
+    pub fn from_o_flags(flags: libc::c_int) -> (Self, libc::c_int) {
```

Comment:
Could use a doc-comment explaining the second return value

Reply:
Added more comments.